miden-tx/src/tests/kernel_tests/test_tx.rs

```diff
@@ -623,15 +621,19 @@ fn test_build_recipient_hash() {
 // FOREIGN PROCEDURE INVOCATION TESTS
 // ================================================================================================
 
-/// Assert that mock account's `get_item` procedure has index 6
+/// Assert that mock account's `get_item` procedure has index 8
 #[test]
 fn test_check_get_item_index() {
```

Comment:
I replaced calling foreign procedures by root in favor of using `procref`. Maybe we don't want this change, but if we do, I'm not sure we need this test anymore.

Reply:
Sounds reasonable to me. I'm not sure if the intent of the test was to use the MAST root deliberately to test something, but perhaps @Fumuran can clarify.

Reply:
Excuse me for the late reply, I missed the notification. 
I think using the `procref` is much better, for some reason I was so fixated on obtaining the procedure's hash, that I forgot that we can just obtain it with `procref`. This procedure is necessary only for checking the correct index of the `get_item` procedure, but since now we use `procref` this test is not needed anymore. I'll remove it in my test refactoring PR.